### ISSUE TITLE
[Feature Request] Add compatibility with safe afk mods, if i get gicked by one of those it tries to reconnect

### ISSUE THREAD

**Issue by hamsterrooo** [Bstn1802/AutoReconnect#62]

**Comment by Bstn1802** [comment]
Good point, I'll definitely look into this, but I currently don't really have the time to work on this mod, still even trying to find the time to update to 1.20.2 🥲

**Comment by hamsterrooo** [comment]
Also I think adding safe afk to this mod would be a better idea, i guess checking if player was damaged/health got lower than **_x_** isn't that hard

**Comment by TechPro424** [comment]
Safe afk mods?

**Comment by Bstn1802** [comment]
Safe afk mods = mods that kick you when you get too much damage or are about to die in general, to prevent exactly that.
Any features in that direction are not planned for this mod and on a second thought I also don't quite understand how those mods would even trigger the automatic reconnect, since those mods should make the player leave the server instead of abruptly ending the connecting with the client ending up on the disconnect screen, right? So actually, I'm confused, please explain @Jamnik-Klembus 🤔
